pkg/security/resolvers/sbom/resolver.go

```diff
@@ -183,7 +187,13 @@ func (r *Resolver) Start(ctx context.Context) {
 				return
 			case sbom := <-r.scannerChan:
 				if err := r.analyzeWorkload(sbom); err != nil {
-					seclog.Errorf("couldn't scan '%s': %v", sbom.ContainerID, err)
+					go func() {
+						if err := retry.Do(func() error {
+							return r.analyzeWorkload(sbom)
+						}, retry.Attempts(maxSBOMGenerationRetries), retry.Delay(20*time.Millisecond)); err != nil {
```

Comment:
I don't know if it could be relevant here but perhaps an error could occur because the `scanQueue` is full. It is the case for container images, I don't know if the host scan might be concerned as well.